ofrak_core/requirements.txt

```diff
@@ -6,7 +6,6 @@ importlib-metadata>=1.4
 intervaltree==3.1.0
 keystone-engine==0.9.2
 lief==0.12.3
-orjson~=3.8.7
```

Comment:
Is removing the `orjson` dependency supposed to be part of this PR?

If anything, we should try and import it, and fall back on the regular `json` library if we can't. But I don't think removing it entirely makes sense.